ambry-network/src/main/java/com.github.ambry.network/PlainTextTransmission.java

```diff
@@ -91,6 +91,7 @@ public boolean write() throws IOException {
         socketChannel.socket().getRemoteSocketAddress(), getConnectionId(), writeTimeMs);
     if (bytesWritten > 0) {
       metrics.plaintextSendTimePerKB.update(writeTimeMs * 1024 / bytesWritten);
```

Comment:
Not your change, but I think we are doing this wrong. One of these variables should be forced to `float`; I suspect this resolves to 0 most of the time - with 300KB written (~ average size we see), even with 60 ms send time, this would resolve to `60 * 1024 / (300 * 1024) == 0`.

Reply:
.update() needs (int val) 

instead of forced to float then int again, I think we can just output microsecond here. 

metrics.plaintextSendTimePerKB.update(writeTimeMs * 1024000 / bytesWritten);

Reply:
microseconds is fine, we just need to name the metric accordingly (`plainTextSendTimeInUsPerKB`?).